libs/shared/modules/src/comment/README.md

```diff
@@ -0,0 +1,183 @@
+# Comments API
```

Comment:
Should this have a copy of the model? I think it'll get outdated easily. Rather have some documentation about how it works together and referencing the relevant models in code?

Reply:
Ah yes, I used this for myself when refactoring the api.